nodepy/twostep_runge_kutta_method.py

```diff
@@ -147,6 +131,94 @@ def stability_matrix(self,z):
         M=np.vstack([L1,[1.,0.]])
         return M
 
+    def __num__(self):
+        """
+        Returns a copy of the method but with floating-point coefficients.
+        This is useful whenever we need to operate numerically without
+        worrying about the representation of the method.
+        """
+        import copy
+        numself = copy.deepcopy(self)
+        if self.A.dtype==object:
+            raise('Not yet Implement')
+
+        return numself
+
+
+    def __step__(self,f,t,u,dt,x=None,estimate_error=False,use_butcher=False):
+        """
+            Take a time step on the ODE u'=f(t,u).
+
+            **Input**:
+                - f  -- function being integrated
+                - t  -- array of previous solution times
+                - u  -- array of previous solution steps (u[i] is the solution at time t[i])
+                - dt -- length of time step to take
+
+            **Output**:
+                - unew -- approximate solution at time t[-1]+dt
+        """
+        import pdb
+        Q = self.Q
+	d_tilde = self.d_tilde.transpose()
+	theta_tilde = self.theta_tilde
+	eta = self.eta
+
+	r = self.r
+	s = Q.shape[0]# why do I need this? (explit methods, first row is all zeros)
+	from nodepy import runge_kutta_method as rk
```

Comment:
I'm not sure what's going on here -- _step__ should take one step with a TSRK method.  Why do you import rk and semidisc?